src/share/poudriere/jail.sh

```diff
@@ -351,13 +356,10 @@ build_and_install_world() {
 		STRINGS=/nxb-bin/usr/bin/strings
 		AWK=/nxb-bin/usr/bin/awk
 		FLEX=/nxb-bin/usr/bin/flex
+		_MAKE_JOBS=-j1
+		MAKE_JOBS_NUMBER=1
```

Comment:
This made into 6b1ddeb7faff240993188dd430a0be9dc8b6d802 and I still think it is not right.